backend/src/main/kotlin/app/ehrenamtskarte/backend/graphql/regions/RegionsQueryService.kt

```diff
@@ -1,118 +0,0 @@
-                        it.dataPrivacyPolicy,
-                        it.activatedForApplication,
-                        it.activatedForCardConfirmationMail,
-                        it.applicationConfirmationMailNoteActivated,
-                        it.applicationConfirmationMailNote,
-                    )
-                }
-            }
-        }
-
-    @GraphQLDescription("Returns region data for specific region.")
-    fun regionByRegionId(regionId: Int): Region =
-        transaction {
-            val regionEntity = RegionsRepository.findRegionById(regionId)
-            Region(
-                regionEntity.id.value,
-                regionEntity.prefix,
-                regionEntity.name,
-                regionEntity.regionIdentifier,
-                regionEntity.dataPrivacyPolicy,
-                regionEntity.activatedForApplication,
-                regionEntity.activatedForCardConfirmationMail,
-                regionEntity.applicationConfirmationMailNoteActivated,
-                regionEntity.applicationConfirmationMailNote,
-            )
-        }
-
-    @GraphQLDescription(
-        "Returns regions by postal code. Works only for the EAK project in which each region has an appropriate regionIdentifier.",
-    )
-    fun regionsByPostalCode(dfe: DataFetchingEnvironment, postalCode: String, project: String): List<Region> =
-        transaction {
-            val projectEntity = ProjectEntity.find { Projects.project eq project }.firstOrNull()
-                ?: throw ProjectNotFoundException(project)
-
-            if (projectEntity.project != EAK_BAYERN_PROJECT) {
-                throw NotEakProjectException()
-            }
-            val regionEntities = dfe.graphQlContext.context.regionIdentifierByPostalCode
-                .filter { pair -> pair.first == postalCode }
-                .mapNotNull { region ->
-                    RegionsRepository.findRegionByRegionIdentifier(region.second, projectEntity.id)
-                }
-                .takeIf { it.isNotEmpty() }
-                ?: throw RegionNotFoundException()
-
-            regionEntities.map {
-                Region(
-                    it.id.value,
-                    it.prefix,
-                    it.name,
-                    it.regionIdentifier,
-                    it.dataPrivacyPolicy,
-                    it.activatedForApplication,
-                    it.activatedForCardConfirmationMail,
-                    it.applicationConfirmationMailNoteActivated,
-                    it.applicationConfirmationMailNote,
-                )
-            }
-        }
-
-    @Deprecated("Deprecated in favor of project specific query", ReplaceWith("regionsInProject"))
-    @GraphQLDescription("Return list of all regions in the eak bayern project.")
-    fun regions(): List<Region> = regionsInProject(DEFAULT_PROJECT)
-
-    @Deprecated("Deprecated in favor of project specific query", ReplaceWith("regionsByIdInProject"))
```

Comment:
I haven't migrated these functions, deprecated since Feb 13, 2023
I believe we shouldn't have any active users who haven't updated the app in 3 years❓